### PR TITLE
ES v5.28 docs

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

**Comment by esethna** [comment]
@amyblais can you help hook up the redirect from the in-product link to this docs page? https://mattermost.atlassian.net/browse/MM-28847

```
https://about.mattermost.com/default-notices/
```

File: source/administration/notices.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+In-Product Notices
+==================
+
+Mattermost is introducing in-product notices to keep users and adminstrators informed of the newest product improvements, features and releases.
```
Comment (amyblais) [inline source/administration/notices.rst:4]:
```suggestion
Mattermost is introducing in-product notices to keep users and administrators informed of the newest product improvements, features and releases.
```

File: source/administration/notices.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+==================
+
+Mattermost is introducing in-product notices to keep users and adminstrators informed of the newest product improvements, features and releases. 
+
+.. contents::
+  :depth: 2
+  :local:
+  :backlinks: entry
+  
+ .. image:: ../images/notices.png
```
Comment (amyblais) [inline source/administration/notices.rst:11]: Image doesn't show.

File: source/administration/notices.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Adminstrator Notices
```
Comment (amyblais) [inline source/administration/notices.rst:13]:
```suggestion
Administrator Notices
```

File: source/administration/notices.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+---------------------
+Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also recieve notices about recommended server configuration options to optimize the user experience of their deployment.
```
Comment (amyblais) [inline source/administration/notices.rst:15]:
```suggestion
Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
```

File: source/administration/notices.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+End User Notices
+-----------------
+End user notices are used to inform users and Admins of new feature enhancements, and when new desktop versions are available.  
+
+End User notices can be disabled in the **System Console > Notices** page. 
+
+
+FAQ
+---
+Are notices enabled by-default?
```
Comment (amyblais) [inline source/administration/notices.rst:28]:
```suggestion
Are notices enabled by default?
```

File: source/administration/notices.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Notices are enabled by-default for all Mattermost users. System Admins may choose to disable administrator or end user notices in the **System Console > Notices** page.
```
Comment (amyblais) [inline source/administration/notices.rst:30]:
```suggestion
Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in the **System Console > Notices** page.
```

File: source/administration/notices.rst [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+Will I still receive notices if my server is air-gapped? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+No, the Mattermost server requires a connection to the internet to recieve notices.
```
Comment (amyblais) [inline source/administration/notices.rst:34]:
```suggestion
No, the Mattermost server requires a connection to the internet to receive notices. 
```

File: source/administration/notices.rst [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Notices are enabled by-default for all Mattermost users. System Admins may choose to disable administrator or end user notices in the **System Console > Notices** page.
+
+Will I still receive notices if my server is air-gapped? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+No, the Mattermost server requires a connection to the internet to recieve notices. 
+
+How often will users receive notices?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Notices will be used to raise awreness of new features as part of our monthly release cadence. Users will only recieve notices that apply to them specifically. For example, if a user is already running the latest desktop version, they will not recieve a desktop upgrade notice.
```
Comment (amyblais) [inline source/administration/notices.rst:38]:
```suggestion
Notices will be used to raise awareness of new features as part of our monthly release cadence. Users will only receive notices that apply to them specifically. For example, if a user is already running the latest desktop app version, they will not receive a desktop upgrade notice.  
```

File: source/help/getting-started/configuring-notifications.rst [hunk 9]
```diff
@@ -12,11 +12,11 @@ Desktop, email, and mobile push notifications notify you of activity in Mattermo
 Desktop Notifications
 -------------------------------------
 
-Desktop notifications are pop-ups that appear in the corner of your main monitor when using Chrome, Firefox, Edge, Safari, or the `Mattermost Desktop app <https://mattermost.com/download/#mattermostApps>`_. You can change the default preference to trigger desktop notifications for all messages sent in channels you're a member of, or turn them off entirely.
+Desktop notifications are pop-ups that appear in the corner of your main monitor when using Chrome, Firefox, Edge, Safari, or the `Mattermost Desktop apps <https://mattermost.com/download/#mattermostApps>`_. You can change the default preference to trigger desktop notifications for all messages sent in channels you're a member of, or turn them off entirely. You can also choose what sound plays when a desktop notification is triggered on supported browsers and the Mattermost Desktop apps.
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst:15]:
```suggestion
Desktop notifications are pop-ups that appear in the corner of your main monitor when using Chrome, Firefox, Edge, Safari, or the `Mattermost Desktop Apps <https://mattermost.com/download/#mattermostApps>`_. You can change the default preference to trigger desktop notifications for all messages sent in channels you're a member of, or turn them off entirely. You can also choose what sound plays when a desktop notification is triggered on supported browsers and the Mattermost Desktop Apps.
```

**Comment by esethna** [comment]
@justinegeffen the image and toc tree isn't appearing for some reason on the notices doc

**Review by amyblais — CHANGES_REQUESTED** [review]

File: source/help/getting-started/configuring-notifications.rst [hunk 10]
```diff
@@ -12,11 +12,11 @@ Desktop, email, and mobile push notifications notify you of activity in Mattermo
...
 
 .. image:: ../../images/desktop_notification.png
 
--  Configure desktop notifications in **Account Settings > Notifications > Desktop notifications > Send desktop notifications**.
+-  Configure desktop notifications in **Account Settings > Notifications > Desktop notifications > Send desktop notifications** and choose your sound selection.
```
Comment (justinegeffen) [inline source/help/getting-started/configuring-notifications.rst:19]:
```suggestion
-  Configure desktop notifications in **Account Settings > Notifications > Desktop Notifications > Send desktop notifications** and choose your sound selection.
```

File: source/administration/notices.rst [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Notices are enabled by-default for all Mattermost users. System Admins may choose to disable administrator or end user notices in the **System Console > Notices** page.
+
+Will I still receive notices if my server is air-gapped? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+No, the Mattermost server requires a connection to the internet to recieve notices. 
+
+How often will users receive notices?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Notices will be used to raise awareness of new features as part of our monthly release cadence. Users will only receive notices that apply to them specifically. For example, if a user is already running the latest desktop app version, they will not receive a desktop upgrade notice.
```
Comment (justinegeffen) [inline source/administration/notices.rst:38]:
```suggestion
Notices will be used to raise awareness of new features as part of our monthly release cadence. Users will only receive notices that specifically apply to them. For example, if a user is already running the latest Desktop App version, they will not receive an upgrade notice. 
```

File: source/administration/notices.rst [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+FAQ
+---
+Are notices enabled by-default?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Notices are enabled by-default for all Mattermost users. System Admins may choose to disable administrator or end user notices in the **System Console > Notices** page.
+
+Will I still receive notices if my server is air-gapped? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+No, the Mattermost server requires a connection to the internet to receive notices.
```
Comment (justinegeffen) [inline source/administration/notices.rst:34]:
```suggestion
No, the Mattermost server requires a connection to the internet to receive notices.
```

File: source/administration/notices.rst [hunk 13]
```diff
@@ -0,0 +1,38 @@
+In-Product Notices
+==================
+
+Mattermost is introducing in-product notices to keep users and administrators informed of the newest product improvements, features and releases.
```
Comment (justinegeffen) [inline source/administration/notices.rst:4]:
```suggestion
Mattermost in-product notices keep users and administrators informed of the newest product improvements, features, and releases.
```

File: source/administration/notices.rst [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+---------------------
+
+Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
+
+Administrator notices can be disabled in the **System Console > Notices** page.
+
+End User Notices
+----------------
+
+End user notices are used to inform users and Admins of new feature enhancements, and when new desktop versions are available. End User notices can be disabled in the **System Console > Notices** page.
```
Comment (justinegeffen) [inline source/administration/notices.rst:23]:
```suggestion
End user notices are used to inform users and Admins of new feature enhancements and when new desktop versions are available. They can be disabled in **System Console > Notices**.
```

File: source/administration/notices.rst [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+FAQ
+---
+
+Are notices enabled by default?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in the **System Console > Notices** page.
```
Comment (justinegeffen) [inline source/administration/notices.rst:31]:
```suggestion
Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in **System Console > Notices**.
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Docs for: https://github.com/mattermost/mattermost-server/pull/15316.

**Comment by amyblais** [comment]
@esethna Will the In-Product Notices config settings be documented? ``AdminNoticesEnabled``, ``UserNoticesEnabled``, ``NoticesURL``, ``NoticesFetchFrequency``, and ``NoticesSkipCache``

**Comment by justinegeffen** [comment]
@esethna, could you add this to the relevant guide's TOC so that it displays on the LHS and I can validate the preview? Thank you!

**Comment by esethna** [comment]
@justinegeffen how do I do that?

**Comment by amyblais** [comment]
@esethna You can update it by navigating to the TOC page, not sure if it would be https://docs.mattermost.com/guides/administrator.html or https://docs.mattermost.com/guides/user.html.

**Comment by esethna** [comment]
@justinegeffen @amyblais done but I accidently committed directly to the v5.28-documentation branch

**Comment by justinegeffen** [comment]
Awesome, thanks @esethna! So it's all rendering perfectly except the image and I'm not sure what the issue is. Here's a screenshot of how it renders in [preview](https://mattermost.github.io/docs/esethna/administration/notices.html) vs the code in the docs. 

Screen Shot 2020-10-13 at 18 41 21

Screen Shot 2020-10-13 at 18 41 28

@amyblais, do you have any idea where the problem is? It all looks like it should work fine.

**Comment by amyblais** [comment]
I'm unsure to be honest, will need to troubleshoot further.

@esethna Did we also want to include the missing config settings https://github.com/mattermost/docs/pull/3968#issuecomment-707177022?

**Comment by justinegeffen** [comment]
I propose removing the screenshot for now so it doesn't block the docs, and we can add it back post-release. @esethna, does that work for you?

**Comment by esethna** [comment]
@justinegeffen that's fine. I'd propose we merge as is as it may work on docs once merged. Let's just make sure to revisit after merge to get the screenshot working

**Comment by esethna** [comment]
@amyblais adding now

**Comment by esethna** [comment]
@amyblais config settings added, we are not documenting NoticesURL, NoticesFetchFrequency, and NoticesSkipCache because they are only used for testing

**Pull request merged**